local.tf

```diff
@@ -43,7 +43,7 @@ locals {
     spot_price                    = ""                          # Cost of spot instance.
     placement_tenancy             = ""                          # The tenancy of the instance. Valid values are "default" or "dedicated".
     root_volume_size              = "100"                       # root volume size of workers instances.
-    root_volume_type              = "gp2"                       # root volume type of workers instances, can be 'standard', 'gp3' (for Launch Template), 'gp2', or 'io1'
+    root_volume_type              = "gp3"                       # root volume type of workers instances, can be "standard", "gp3", "gp2", or "io1"
```

Comment:
Why change this? It was recently changed in the opposite direction by https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1208

Reply:
Good catch. This was due to my rebase off of main. Fixing.

Reply:
reverted this change.